homeassistant/components/komfovent/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""The Komfovent integration."""
+from __future__ import annotations
+
+from typing import cast
+
+import komfovent_api
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+
+PLATFORMS: list[Platform] = [Platform.CLIMATE]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Komfovent from a config entry."""
+    host = cast(str, entry.data[CONF_HOST])
+    username = cast(str, entry.data[CONF_USERNAME])
+    password = cast(str, entry.data[CONF_PASSWORD])
```

Comment:
I still don't like these casts personally, I asked this to someone else for a second opinion

Reply:
```suggestion
    host = entry.data[CONF_HOST]
    username = entry.data[CONF_USERNAME]
    password = entry.data[CONF_PASSWORD]
```
I got a response and we think we can remove the casts. If you really want complete type safety, we would suggest using a TypedDict. But I think that could be part of a follow-up PR.